iamproperty/iamproperty.github.io
# Sub nav  examples

## Summary of Changes:
1. Update the CSS
2. Update examples
3. Add a new bit of JS

## Rationale
Update the nav component to handle sub navigation.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/nav then review the 'Sub navigation' section

## Ticket(s)
DS-46

## Notes
Still need to develop how to add search functionality to the navs.
